cfme/tests/integration/test_openldap_external_auth.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+import pytest
+
+from cfme.configure.access_control import Group, User
+from cfme import Credential, login
+from utils.conf import cfme_data
+from utils.providers import setup_a_provider
+
+
+@pytest.fixture(scope="module")
+def setup_first_provider():
+    setup_a_provider(validate=True, check_existing=True)
+
+
+@pytest.fixture()
+def group():
+    data = cfme_data.get("openldap_domain1", {})
+    if not data:
+        pytest.skip("No openldap_test section in yaml")
+    credentials = Credential(
+        principal=data["username"],
+        secret=data["password"],
+    )
+    return Group(description=data['group_name'], role="EvmRole-user",
+                 user_to_lookup=data["username"], ldap_credentials=credentials)
+
+
+@pytest.mark.tier(3)
+@pytest.mark.parametrize(
+    "add_from_ldap", [False, True],
+    ids=['create', 'add_group_from_ldap_lookup'])
+def test_openldap_ext_auth(request, group, add_from_ldap, configure_openldap_external_auth_mode):
+    data = cfme_data.get("openldap_domain1", {})
+    if add_from_ldap:
+        group.add_group_from_ext_auth_lookup()
+    else:
+        group.create()
+    request.addfinalizer(group.delete)
+    credentials = Credential(
+        principal=data["username"],
+        secret=data["password"],
+        verify_secret=data["password"],
+    )
+    user = User(name=data["username"], credential=credentials)
+    request.addfinalizer(user.delete)
```

Comment:
@psav  group associated to the user is added/retrieved first. then try authenticating that user from cfme webui (only webui for now).
Yes, the data for group_name associated to the user is in the yaml.